vj4/ui/pages/discussion_main.page.styl

```diff
@@ -1,5 +1,6 @@
 .page--discussion_main,
-.page--discussion_node
+.page--discussion_node,
+.page--user_detail
```

Comment:
I suggest to move discussion__item into a component instead of assigning more than one page in a page's style.